New-TieredStorageSpace.ps1

```diff
@@ -0,0 +1,167 @@
+# RUN AS ADMINISTRATOR
+# https://nils.schimmelmann.us/post/153541254987/intel-smart-response-technology-vs-windows-10
+#Tested with one SSD and two HDD
+#Requires -RunAsAdministrator
+
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $False, ValueFromPipeline = $True)]
+    [string[]]
+    $ConfigFile
+)
+
+Begin{
+    #Initialize Variables
+    $Script:StorageSpacesParams = @()
+    #Makes $PhysicalDisks a hashtable for easier management.
+    $PhysicalDisks = @()
+    $Script:TieredDiskName
+    
+
+    function defaultPrompt {
+        if ($null -eq $ConfigFile) {
+            $DefaultPrompt = Read-Host = "Would you like to load the default values to the config file? (Y/N)" 
```

Comment:
Is this going to save defaults to a config file or load them from a config file. I'm not sure from the prompt.